core/views.py

```diff
@@ -49,7 +49,17 @@ def register(request):
                 new_user.profile.city = form.data['city']
                 new_user.profile.country = form.data['country']
                 new_user.save()
-                return redirect('/')
+                authenticated = authenticate(
+                    username=new_user.username,
+                    password=new_user.password,
```

Comment:
This is so wrong. Please read: https://docs.djangoproject.com/en/1.11/ref/contrib/auth/#django.contrib.auth.models.User.set_password